### PR TITLE
Add combinations including business_interruption

### PR DESCRIPTION
**Pull request by ptormene**
This fixes the case highlighted by SURA with structural+nonstructural+business_interruption.
I've added also many other possible combinations, but I suppose many others should be added to obtain an extensive set.

### CODE REVIEW

**Comment by ptormene** [comment]
Thanks @raoanirudh. In oqvalidate.py it should be sufficient.
I remain a little concerned about scientific.py, where `LOSSTYPE` also contains:
`'''occupants occupants_day occupants_night occupants_transit structural+nonstructural_ins structural+contents_ins nonstructural+contents_ins structural+nonstructural+contents_ins structural_ins nonstructural_ins reinsurance claim area number'''.split()`

File: openquake/commonlib/oqvalidation.py [hunk 1]
```diff
@@ -838,6 +839,12 @@
                    'event_based_damage',
                    'scenario_damage']
 
+LOSS_TYPES = [
+    'structural', 'nonstructural', 'contents', 'business_interruption']
+LOSS_TYPE_COMBINATIONS = [
+    '+'.join(s) for l_idx in range(len(LOSS_TYPES))
+    for s in itertools.combinations(LOSS_TYPES, l_idx + 1)]
+
```
Comment (micheles) [inline openquake/commonlib/oqvalidation.py:847]: Please change the name from LOSS_TYPES to COST_TYPES.

File: openquake/risklib/scientific.py [hunk 2]
```diff
@@ -50,7 +50,13 @@
...
 structural+nonstructural+contents_ins
 structural_ins nonstructural_ins
-reinsurance claim area number'''.split())
+reinsurance claim area number
+structural+business_interruption nonstructural+business_interruption
+contents+business_interruption
+structural+nonstructural+business_interruption
+structural+contents+business_interruption
+nonstructural+contents+business_interruption
+structural+nonstructural+contents+business_interruption'''.split())
```
Comment (micheles) [inline openquake/risklib/scientific.py:59]: We need to add exactly the same loss types but with an `_ins` suffix.

File: openquake/commonlib/oqvalidation.py (hunk 1 above)
Comment (ptormene) [reply openquake/commonlib/oqvalidation.py:847]: Done

File: openquake/risklib/scientific.py (hunk 2 above)
Comment (ptormene) [reply openquake/risklib/scientific.py]: Done

**Review by micheles — APPROVED** [review]

**Pull request merged**